### DISCUSSION TITLE
Jingle Playlist

### DISCUSSION THREAD

**Discussion by Rafaelrds2017** [AzuraCast/AzuraCast#4653] [Q&A]
Playlist does not play all audio at once.

The Playlist has 5 commercials, w should play all 5 commercials in sequence, and at the scheduled time. but that doesn't happen. Sometimes it plays only the first audio.

The Commercial Playlist or Jingle must obey the option to play the entire Playlist at the scheduled time.

can someone help me with this? will it be a bug or improper configuration?

**Comment by NixAshton** [comment]
Are you still having this issue Rafael?

**Reply by Rafaelrds2017** [reply]
yes

**Reply by NixAshton** [reply]
Can you confirm these points:

Commercial playlist is set to:

- Basic info > Playlist type - "General Rotation"
- Basic info > "Avoid Duplicate Artists/Titles" - unticked
- Source > Song-Based Playlist - "Sequential"
- Schedule > Relative time values set
- Schedule > "Loop once" ticked
- Station time zone set correctly

?

**Comment by Rafaelrds2017** [comment]
Can you confirm these points:

Commercial playlist is set to:

  *   Basic info > Playlist type - "General Rotation"
  *   Ok
  *   Source > Song-Based Playlist - "Sequential"
  *   Ok
  *   Schedule > Relative time values set
  *   Ok
  *   Schedule > "Loop once" ticked
  *   Ok
  *   Station time zone set correctly

________________________________

**Comment by Rafaelrds2017** [comment]
yes, all configured. according to the attached image
________________________________
De: Nix Ashton ***@***.***>
Enviado: terça-feira, 12 de outubro de 2021 10:07
Para: AzuraCast/AzuraCast
Cc: Rafael Rodrigues; Author
Assunto: Re: [AzuraCast/AzuraCast] Jingle Playlist (Discussion #4653)


Can you confirm these points:

Commercial playlist is set to:

  *   Basic info > Playlist type - "General Rotation"
  *   Source > Song-Based Playlist - "Sequential"
  *   Schedule > Relative time values set
  *   Schedule > "Loop once" ticked
  *   Station time zone set correctly

?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/AzuraCast/AzuraCast/discussions/4653#discussioncomment-1464695>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AHS2JT2AUDADDY3ITHGK6UTUGQXJ3ANCNFSM5FPAUBBA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by Rafaelrds2017** [comment]
![Screenshot_20211012-102148_Chrome](https://user-images.githubusercontent.com/31827151/136964703-66f4f27b-306d-4fad-88bc-ce5f1db1191f.jpg)

**Comment by Rafaelrds2017** [comment]
![Screenshot_20211012-102221_Chrome](https://user-images.githubusercontent.com/31827151/136964750-1e39f5e4-38b3-4404-8780-e41d71803500.jpg)

**Comment by Rafaelrds2017** [comment]
![Screenshot_20211012-102239_Chrome](https://user-images.githubusercontent.com/31827151/136964785-f8ac3871-f4e2-4025-b414-21b6610b0a08.jpg)

**Comment by Rafaelrds2017** [comment]
![Uploading Screenshot_20211012-102257_Chrome.jpg…]()

**Comment by NixAshton** [comment]
Peso da playlist - 25? Make any difference?

**Comment by Rafaelrds2017** [comment]
I will put weight 25 to test

Obter o Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Rafaelrds2017** [comment]
It didn't work completely. Only plays 1 audio from Playlist.

should play the 5 commercial audio. but only reproduced 1

Obter o Outlook para Android<https://aka.ms/AAb9ysg>
________________________________
